internal/core/config.go

```diff
@@ -208,13 +208,22 @@ func (cluster ClusterConfiguration) validateConfig() (errs errext.ErrorSet) {
 	}
 
 	// NOTE: Liquids[].FixedCapacityConfiguration and Liquids[].PrometheusCapacityConfiguration are optional
+	occupiedConversionIdentifiers := make([]string, 0)
```

Comment:
A zero-length allocation is always worthless (unless you want to ensure that the slice never serializes as `null` instead of `[]` in JSON). A nil slice behaves the same as an empty one, but does not need an allocation.
```suggestion
	var occupiedConversionIdentifiers []string
```